src/components/launcherdbus.cpp

```diff
@@ -27,18 +27,31 @@
 LauncherDBus::LauncherDBus(LauncherModel *model)
     : QObject(model)
     , QDBusContext()
-    , m_model(model)
 {
     QDBusConnection dbus = QDBusConnection::sessionBus();
     dbus.registerObject(LIPSTICK_DBUS_LAUNCHER_MODEL_PATH,
             this, QDBusConnection::ExportAllSlots |
             QDBusConnection::ExportAllSignals);
```

Comment:
Only one object at a time can be registered to a path, which doesn't work out too well with multiple instances of LauncherModel each owning an instance of LauncherDBus.

It does seem to me that this should be listening for signals emitted from a service provided by the store client rather than the other way around.  But I'll settle for a backward compatible solution rather than start reinventing things.